src/lib/services/client-metrics/client-metrics.test.ts

```diff
@@ -252,11 +295,11 @@ test('should have correct values for lastMinute', () => {
     let c = metrics.getTogglesMetrics();
     expect(c.lastMinute.toggle).toEqual({ yes: 20, no: 20 });
 
-    jest.advanceTimersByTime(10 * 1000);
+    jest.advanceTimersByTime(10_000);
```

Comment:
yep, having played with it a bit, I think anything below 1 minute is better written as a constant with underscores than either `10 * 1000` or `secondsToMilliseconds(10)`